lib/msal-browser/docs/shr-server-nonce.md

```diff
@@ -0,0 +1,263 @@
+# SHR Server Nonce
+
+As an enhancement of Access Token Proof-of-Possesion, MSAL Browser provides a way to insert a server-generated signed timestamp (a.k.a **server nonce**) into a `Signed HTTP Request`, also known as a `PoP Token`. These server generated nonce can be added to any token request that uses the `POP` authentication scheme.
+
+Given that [MSAL does not cache](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/access-token-proof-of-possession.md#bound-access-token) `Signed HTTP Requests`, server nonces will **not** be cached either. This means that the server nonce must be passed into every `acquireTokenSilent` call in order for it to be added to the resulting `SignedHttpRequest` object.
+
+Once the `Signed HTTP Request` is sent to the resource server as a `PoP Token`, the resource server is responsible for validating the signed payload as well as extracting and validating the `shrNonce`.
+
+## Usage
+
+Once [acquired](#acquiring-a-server-nonce), server-generated nonces for SHRs can be passed into any token request object as `shrNonce`. It is important to note that the SHR server nonce is an extension of the Access Token Proof-of-Possession scheme, so it will only be used when the token request's `authenticationScheme` is set to `POP`.
+
+### Acquire Token Redirect Request Example
+
+```javascript
+const popTokenRequest = {
+    scopes: ["User.Read"],
+    authenticationScheme: msal.AuthenticationScheme.POP,
+    resourceRequestMethod: "POST",
+    resourceRequestUri: "YOUR_RESOURCE_ENDPOINT",
+    shrNonce: "eyJhbGciOiJIUzI1NiIsImtpZCI6IktJRCIsInR5cCI6IkpXVCJ9.eyJ0cyI6IjE2MjU2NzI1MjkifQ.rA5ho63Lbdwo8eqZ_gUtQxY3HaseL0InIVwdgf7L_fc" // Sample Base64URL encoded server nonce value
+}
+```
+
+Once the request has been configured and `POP` is set as the `authenticationScheme`, it can be sent into the `acquireTokenRedirect` MSAL Browser API.
```

Comment:
To avoid confusion let's not make this specific to acquireTokenRedirect
```suggestion
Once the request has been configured and `POP` is set as the `authenticationScheme`, it can be passed to any loginXXX or acquireTokenXXX API:
```